tests/commands/dev/test_call.py

```diff
@@ -38,14 +42,36 @@ def verify_app_tools(self, app):
         super().verify_app_tools(app=app)
         self.actions.append(("verify-app-tools", app.app_name))
 
-    def install_dev_requirements(self, app, **kwargs):
-        self.actions.append(("dev_requirements", app.app_name, kwargs))
+    def get_venv_context(self, appname, isolated=False):
+        self.actions.append(("get-venv-context", appname, isolated))
```

Comment:
This test piece hasn't been updated to reflect the change in the underlying method.